public/usage-examples/graphics/current_clip/current-clip-1-simple-oop.cs

```diff
@@ -0,0 +1,24 @@
+using SplashKitSDK;
+
+public class Program
+{
+    public static void Main()
+    {   
+        // Open a window with the title "Current Clip" and dimensions
+        Window window= SplashKit.OpenWindow("Current Clip", 800, 600);
+        
+        // Push a clipping area
+        Rectangle rectangle = SplashKit.CurrentClip();
```

Comment:
Add window in CurrentClip() -> CurrentClip(window), otherwise it is no need to create Window window.